docs/Doxyfile_include_for_windows

```diff
@@ -0,0 +1,16 @@
+# This file is intended to be included in the Doxyfile config for Windows.
+# It contains configuration settings that are incompatible with Linux.
+
+# The FILTER_PATTERNS tag can be used to specify filters on a per file pattern
+# basis. Doxygen will compare the file name with each pattern and apply the
+# filter if there is a match. The filters are a list of the form: pattern=filter
+# (like *.cpp=my_cpp_filter). See INPUT_FILTER for further information on how
+# filters are used. If the FILTER_PATTERNS tag is empty or if none of the
+# patterns match the file name, INPUT_FILTER is applied.
+#
+# Note that we currently have to use a patched version of doxypypy.py because of
+# new syntax in python version 3.8.
+# See https://github.com/Feneric/doxypypy/issues/70
+# The patched version is stored in the current directory.
+
+FILTER_PATTERNS      = *.py=py_filter
```

Comment:
these files (linux, windows) are the same. I'd prefer if we can keep a single, platform independent version